spec/index.md

```diff
@@ -465,6 +463,29 @@ Here are examples of the comparison operators being used.
 11 <= 11
 ```
 
+### 4.2.4 `revert` expression
```

Comment:
I think we should refer to `revert` as a statement

Reply:
Ah, you are absolutely right. Will change.